cmd/kubeadm/app/apis/kubeadm/v1alpha1/upgrade_test.go

```diff
@@ -17,37 +17,11 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"io/ioutil"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/serializer"
 )
 
-const test196 = "testdata/kubeadm196.yaml"
```

Comment:
Where'd this test end up?

Reply:
Ugh. I moved it to `testdata` but somehow I didn't commit it. I'll send a non-critical PR with that test later